src/helpers/fileHelper.ts

```diff
@@ -1,5 +1,6 @@
 import * as mime from "mime/lite";
 import { IFile } from "../interfaces/IFile";
+import { picsum } from "@/helpers/picsum";
 
 // Howler (audio player) requires the following extensions:
 // "mp3", "mpeg", "opus", "ogg", "oga", "wav", "aac", "caf", "m4a", "m4b",
```

Comment:
Do we check supported file formats somewhere? That is to say: is this list already somewhere in the code?
In this case, this comment is arguably redundant. If not, it might make sense to explicitly check.

Reply:
I think you refer to line 6, which is not part of this PR, but it could be made into an issue